gravitee-apim-portal-webui-next/src/app/api-details/api-details.component.html

```diff
@@ -55,6 +55,17 @@
         i18n="@@apiDocumentation"
         >Documentation</a
       >
+      @if (isUserAuthenticated) {
+        <a
+          mat-tab-link
+          [routerLink]="['.', 'subscriptions']"
+          routerLinkActive
+          #docActive="routerLinkActive"
```

Comment:
Make sure to use a different identifier than `#docActive` since it is used for the doc tab (above).

Reply:
Done